platform/podjs/src/browserPod.ts

```diff
@@ -544,6 +624,10 @@ class BrowserEndpoint implements Endpoint {
     }
 }
 
+/**
+ * Create a random UUID
+ * @returns A string of length 36.
```

Comment:
Can you please be bit more precise here? What's the usage of that UUID? Maybe mention the format?